lib/page.js

```diff
@@ -12,7 +12,7 @@ function test (opts) {
           return Promise.resolve()
             .then(() => {
               if (opts.scroll || opts.reporter === 'fps') {
-                return session.execute('function f () { window.scrollBy(0,5); window.requestAnimationFrame(f); } window.requestAnimationFrame(f);');
+                return session.execute('dir=-1;function f () { window.scrollBy(0,dir*=-1); window.requestAnimationFrame(f); } window.requestAnimationFrame(f);');
```

Comment:
Actually, I was thinking of explicitly putting it into the window object, or somewhere, to allow calling clearAnimationFrame() after, and run other scripts to calculate the other metrics...